metadata.rb

```diff
@@ -4,7 +4,7 @@
 maintainer_email 'klishinm@vmware.com'
 license 'Apache-2.0'
 description 'Installs and configures RabbitMQ server'
-version '5.10.0'
+version '5.10.1'
```

Comment:
Please never change project version in a PR. This is up to the maintainers to decide when to do it.

Reply:
@michaelklishin - Sorry about the metadata.rb file. I will keep in mind of it in future. For now, I have deleted the metadata.rb file from the pull request. Do you need me to delete this PR and submit a new one? Or is just deleting the metadata,rb file ok for now? Please let me know. Thanks !